owncloudApp/src/main/java/com/owncloud/android/presentation/files/createfolder/CreateFolderDialogFragment.kt

```diff
@@ -82,12 +87,48 @@ class CreateFolderDialogFragment : DialogFragment() {
         alertDialog.setOnShowListener {
             val okButton = alertDialog.getButton(AlertDialog.BUTTON_POSITIVE)
             okButton.isEnabled = isButtonEnabled
+
+            okButton.setOnClickListener {
+                var fileName: String = inputText.text.toString()
+                if (fileName.isEmpty()) {
+                    error = getString(R.string.uploader_upload_text_dialog_filename_error_empty)
+                } else {
+                    null
+                }
+                if (error != null) {
+                    inputLayout.isErrorEnabled = true
+                    inputLayout.error = error
+                } else {
```

Comment:
In theory, this `if` won't be executed ever as well. As we changed the concept of the solution, now we'll show the error in real time (just when the forbidden character is written or the filename exceeds the maximum length) and in that moment, the OK button will be disabled. This means that the OK button will only be clickable when the filename is correct, and in consequence, it will always execute what we have inside the `else` now

Reply:
Done